<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Boundaries</class>
 <widget class="QDialog" name="Boundaries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>231</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Canny_Page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>379</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Canny</string>
    </attribute>
    <widget class="QLabel" name="threshold1Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="threshold2Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold 2</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Threshold1DoubleSpinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Threshold2DoubleSpinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="apertureSizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Aperture
  Size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="apertureIntSizeSpinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l2GradientLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>L2gradient</string>
     </property>
    </widget>
    <widget class="QComboBox" name="l2GradientcomboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>True</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>False</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>379</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="cannyShowLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>571</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
